airflow/api/common/proto/variable.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
```

Comment:
Sorry, I meant the compiled IDL. We need to commit the compiled IDL to the repo as well.